js/repl.js

```diff
@@ -18,9 +18,37 @@ function createEditor(elem) {
   return editor;
 }
 
+var optionsConfig = [
+  {
+    type: "string",
+    name: "mathRandomSeed",
+    defaultVal: "",
+    description: "If you want Prepack to evaluate Math.random() calls, please provide a seed."
+  },
+  {
+    type: "boolean",
+    name: "inlineExpressions",
+    defaultVal: false,
```

Comment:
I would love to see true be the default value here.

Reply:
+1. That would fix #1007!

Reply:
done